src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/WindowsRuntime/EventArgsMarshalerSupport.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.StubHelpers;
+
+namespace Internal.Runtime.InteropServices.WindowsRuntime
+{
+    public static class EventArgsMarshalerSupport
```

Comment:
This is defined as a `public` class/function as such we should probably have some minimal documentation with it. At least something that documents the what the arguments mean. That might be obvious if the function's use was at least explained.

Reply:
File removed.